cypress/support/commands/action.js

```diff
@@ -2,13 +2,26 @@ function executeHeaderAction(actionName) {
   /**
    * Only specific windows can have actions. They match one of the following urls:
    *
-   * https://dev586.metasfresh.com/window/123?viewId=123-o&page=1
-   * https://dev586.metasfresh.com/window/123/2156425
+   * https://dev586.metasfresh.com/window/123?viewId=123-o&page=1 - list view
+   *    - in this case also '.table-flex-wrapper' should exist
+   * https://dev586.metasfresh.com/window/123/2156425 - single view
+   *    - in this case also '.panel' should exist
    *
    * This match is needed because cypress is so fast that it may press the action button before any viewId is available, and the system will error out.
    */
   cy.url().should('matches', new RegExp(`window/[0-9]+(/[0-9]+|.*viewId=)`));
```

Comment:
Won't this throw an error for any page that doesn't match the regexp ?

Reply:
When we come up with the regex, we said that only some pages are supposed to have the actions run on them. 
Anyway, if there are more pages which need this, we can improve the regex at that point.

Reply:
oh, right. This won't be called unless you're actually trying to run an action. Then it's fine.